### ISSUE TITLE
IgnoredPackages does not work

### ISSUE THREAD

**Issue by dermeister0** [digitalcoyote/NuGetDefense#42]
**Describe the bug**
The tool does not exclude the listed packages.

**To Reproduce**
```
    "IgnoredPackages": [
      {
        "Id": "NugetDefense",
        "Version": "1.0.8.0"
      },
      {
        "Id": "NLog",
        "Version": "4.7.5"
      }
    ],
```

or
```
    "IgnoredPackages": [
      {
        "Id": "NugetDefense",
        "Version": "1.0.8.0"
      },
      {
        "Id": "NLog"
      }
    ],
```

```
error : 1 vulnerabilities found for NLog @ 4.7.5
```

**Expected behavior**
The package should be ignored.

**Screenshots**

**Tools (please complete the following information):**
 - IDE: VS2019
 - OS: Windows 10 Update 2004

**Additional context**
The IgnoredPackages property is not used at all.

**Comment by digitalcoyote** [comment]
I'm looking to do a release tomorrow. I'll try to get this fixed by then. I felt something looked off in that, but I didn't have a scenario where I wanted to ignore s package to test with. 

I'll be adding a test for this as well.

**Comment by digitalcoyote** [comment]
1.0.14-pre0001 should be published soon and contains a fix for this, I'm going to try to squeeze in one more issue before the final release

**Comment by digitalcoyote** [comment]
Should be fixed in [v1.0.14](https://github.com/digitalcoyote/NuGetDefense/releases/tag/v1.0.14). I encourage you to reopen this if this build does not fix the issue.
